### PR TITLE
feat: add change entity action

### PR DESCRIPTION
**Pull request by davidsoderberg**
### What change does this PR introduce?

Adds what action the change will do when promoted.

Screenshot 2023-09-28 at 13 24 20
Screenshot 2023-09-28 at 13 25 00
Screenshot 2023-09-28 at 13 25 53
Screenshot 2023-09-28 at 13 38 18
As shown here old changes will not show anything.

### CODE REVIEW

File: apps/api/migrations/changes-migration.ts
Comment (p-fernandez) [inline apps/api/migrations/changes-migration.ts:1]: Line 91 super useful console log 😂

File: apps/api/src/app/change/usecases/create-change/create-change.usecase.ts [hunk 1]
```diff
@@ -30,6 +31,10 @@ export class CreateChange {
     if (change) {
       change.change = changePayload;
 
+      if (command.action === ChangeEntityActionEnum.DELETE) {
+        change.action = command.action;
```
Comment (p-fernandez) [inline apps/api/src/app/change/usecases/create-change/create-change.usecase.ts:35]: We are only showing create and delete changes?

Comment (davidsoderberg) [reply apps/api/src/app/change/usecases/create-change/create-change.usecase.ts:35]: No we show update as well but if a change will be updated instead and it is a delete we update the action to delete.

File: apps/api/migrations/changes-migration.ts
Comment (davidsoderberg) [reply apps/api/migrations/changes-migration.ts:1]: It creates a blank line ;)

File: apps/api/src/app/change/usecases/create-change/create-change.usecase.ts (hunk 1 above)
Comment (davidsoderberg) [reply apps/api/src/app/change/usecases/create-change/create-change.usecase.ts:35]: It is setting it to update here https://github.com/novuhq/novu/pull/4224/files#diff-475c37ecc04947daa8753f461702772f7785813599e76791f8a40c7b16781fb4R133

**Review by p-fernandez — APPROVED** [review]
🌟

**Comment by scopsy** [comment]
@davidsoderberg can we merge conflicts and then merge the branch to next?

**Comment by davidsoderberg** [comment]
> @davidsoderberg can we merge conflicts and then merge the branch to next?

I have fixed the conflicts but I think it needs to be updated in multiple spots because of new changes since this pr was created.

**Comment by scopsy** [comment]
Closing due to inactivity

**Pull request closed without merging**